src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -1750,33 +1752,36 @@ public static synchronized Registration register(Registration registration) {
               copyShallowFrom(registration.service, CodecUtils.fromJson(registration.getState(), Class.forName(fullTypeName)));
             }
           } catch (ClassNotFoundException classNotFoundException) {
-            log.error(String.format("Unknown service class for %s@%s: %s", registration.getName(),
-                    registration.getId(), registration.getTypeKey()), classNotFoundException);
+            log.error(String.format("Unknown service class for %s@%s: %s", registration.getName(), registration.getId(), registration.getTypeKey()), classNotFoundException);

```

Comment:
One line here as well

Reply:
same - ultimately, I'd be fine with refining the definition .. but that should be done in another pr over the entire repo